### PR TITLE
Router API functionality

### PR DESCRIPTION
**Pull request by jerith**
This is the precursor (in `develop`) to #562 (in `develop-frontend`).

### CODE REVIEW

File: go/vumitools/account/models.py [hunk 1]
```diff
@@ -168,6 +168,17 @@ def remove_conversation(self, conv):
...
+    def remove_router(self, router):
+        """Remove all entries linking to or from a given router.
+
+        Useful when archiving a router to ensure it is no longer present in the
+        routing table.
+        """
+        self.remove_connector(str(GoConnector.for_router(
+            router.router_type, router.key, GoConnector.INBOUND)))
+        self.remove_connector(str(GoConnector.for_router(
+            router.router_type, router.key, GoConnector.OUTBOUND)))
```
Comment (hodgestar) [inline go/vumitools/account/models.py:180]: Could we get a unit test for this? Doesn't have to be too extensive since it just calls remove_connector.

**Comment by hodgestar** [comment]
:+1: after a test for the remove_router helper method is added.

File: go/vumitools/account/models.py (hunk 1 above)
Comment (jerith) [inline go/vumitools/account/models.py:180]: Whoops, I thought I was missing a test somewhere.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**